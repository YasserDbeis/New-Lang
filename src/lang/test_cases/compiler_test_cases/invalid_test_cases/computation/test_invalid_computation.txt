func main() 
{
    int x = 2.0 + true;
}