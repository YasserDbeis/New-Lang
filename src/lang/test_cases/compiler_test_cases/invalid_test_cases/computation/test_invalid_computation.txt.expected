RUNTIME ERROR
Error occurred in Execution
Operator invalid for operands
Operation is not valid
-1